src/types.rs

```diff
@@ -831,6 +831,11 @@ impl Context {
     /// };
     ///
     /// assert_eq!(example_context.display(&sig), "A([!] x_)".to_string());
+    ///
+    /// let context = parse_context(&mut sig, "[!] A CONS(SUCC(SUCC(ZERO)) CONS(SUCC(ZERO) CONS(ZERO NIL)))")
+    ///     .expect("parse of [!] A CONS(SUCC(SUCC(ZERO)) CONS(SUCC(ZERO) CONS(ZERO NIL)))") ;
+    /// 
+    /// assert_eq!(context.pretty(&sig), "[!] A [2, 1, 0]"); 
```

Comment:
Shouldn't this be

```rust
assert_eq!(context.display(&sig), ...);
```

to show the differences between `display` and `pretty`?